pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1068,124 +1300,91 @@ def _display_objects(obj_type, object_repo, namespace, cmt_begin, cmt_end,
             :class:`pywbem_mock:InstanceWriteProvider`.  The methods in this
             subclass override the corresponding methods in
             InstanceWriteProvider. The method call parameters must be the
             same as the defult method in InstanceWriteProvider and it must
             return data in the same format if the default method returns data.
-        """  # noqa: E501
-        # pylint: enable=line-too-long
-
-        provider_types = ('instance', 'method')
-
-        if provider_type == 'instance':
-            if not issubclass(provider, InstanceWriteProvider):
-                raise TypeError(
-                    _format("provider argument {0!A} is not a "
-                            "valid subclass of InstanceWriteProvider. ",
-                            provider))
-        elif provider_type == 'method':
-            if not issubclass(provider, MethodProvider):
-                raise TypeError(
-                    _format("provider argument {0!A} is not a "
-                            "valid subclass of MethodProvider. ",
-                            provider))
-        else:
-            raise ValueError(
-                _format("provider_type argument {0!A} is not a valid provider "
-                        "type. Valid provider types are {1!A}.",
-                        provider_type, provider_types))
 
-        if classnames is None:
-            raise ValueError(
-                _format('classnames argument must be string '
-                        'or list of strings. None not allowed.'))
+          namespaces (:term:`string` or :class:`py:list` of :term:`string`):
+            Namespace or namespaces for which the provider is to be registered.
 
-        if not namespaces:
-            namespaces = [self.default_namespace]
+            If `None`, the default namespace of the connection will be used.
 
-            if not isinstance(namespaces, (list, tuple, six.string_types)):
-                raise TypeError(
-                    _format('Namespace argument {0|A} must be a string or '
-                            'list/tuple but is {1}',
-                            namespaces, type(namespaces)))
+          schema_pragma_files  (:term:`string` or :class:`py:list` of :term:`string`):
+            File paths defining a schema pragma file MOF for the set of CIM
+            classes that make up a schema such as the DMTF schema. These files
+            must contain include pragams defining the file location of the
+            classes to be compiled for the defined provider and for any
+            dependencies required to compile those classes.  The directory
+            containing each pragma file is passed to the MOF compiler as the
+            search path for compile dependencies.
 
-            if isinstance(namespaces, six.string_types):
-                namespaces = [namespaces]
+            See :class:`pywbem.MofCompiler` for more information on the
```

Comment:
Fix the class name in this reference, it is `MOFCompiler`

Reply:
Found two of them. DONE